components/CommentReactionPicker.js

```diff
@@ -96,57 +92,54 @@ const CommentReactionPicker = ({ comment, reactions }) => {
                 ref={ref}
               >
                 <InsertEmoticon size="1.2em" />
-              </StyledLink>
+              </StyledButton>
             )}
           </Reference>
           {open && (
             <Popper placement="bottom">
               {({ placement, ref, style }) => (
-                <Container
-                  border="0.1px solid #dadada"
+                <StyledCard
+                  boxShadow="-2px -1px 3px 0px #e8e8e8"
+                  zIndex={9999}
                   data-placement={placement}
                   ref={ref}
-                  style={{
-                    backgroundColor: 'white',
-                    zIndex: 9999,
```

Comment:
@Betree : Removing this zIndex; did you check whether the final comment on a conversation when you click on the emoji addition button whether the popover overlays on top of the `InlineEditField`. Notice that the [InlineEditField has a container which has zindex=2](https://github.com/opencollective/opencollective-frontend/blob/6286583a7199a78fd291c735ebcc01e5c3a9307f/components/InlineEditField.js#L176). :smiley_cat:

Reply:
It's not removed, only moved to the component's props to take advantage of styled-components

![image](https://user-images.githubusercontent.com/1556356/84480382-bf217680-ac94-11ea-954e-a1bf25d78760.png)

Reply:
Ops, I missed it. Sorry. :+1: